index.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 
     if (emberVersion.lt('2.16.0-alpha.1')) {
       this.import('vendor/ember-router-service-polyfill/index.js');
-    } else if (this.parent === this.project && !hasBeenWarned){
+    } else if (this.parent === this.project && !hasBeenWarned && this.project.isEmberCLIAddon()){
```

Comment:
Hmm, wouldn't it be `!this.project.isEmberCLIAddon()`?  I think we want it to warn when this addon is directly included in apps but not addons...

Reply:
Right..